dojo/tools/risk_recon/parser.py

```diff
@@ -20,48 +19,75 @@ def get_findings(self, filename, test):
         if filename:
             tree = filename.read()
             try:
-                data = json.loads(str(tree, 'utf-8'))
-            except:
+                data = json.loads(str(tree, "utf-8"))
+            except BaseException:
```

Comment:
Change to `Exception`

Reply:
Changes apply in commit https://github.com/DefectDojo/django-DefectDojo/pull/8305/commits/5cc690eda741b1f58c6e5d39075007108035d7a7